testing/code/test_source_multiline_block.py

```diff
@@ -1,4 +1,3 @@
-# flake8: noqa
```

Comment:
actually, going to revert this line -- we don't use autopep8 any more so I can merge this back into the other test -- PR incoming for that!